File: leetcode/1929_ConcatenationofArray.java
class Solution {
    public int[] getConcatenation(int[] nums) {

        int newLen = nums.length * 2;
        int[] longNums = new int[newLen];

        for (int i = 0; i < newLen; i++){
            
            longNums[i] = nums[i % nums.length];
        }

        return longNums;
    }
}

File: leetcode/1929_ConcatenationofArray.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:

        '''
        newLen = len(nums) * 2

        resultList = []

        for i in range(newLen):

            resultList.append(nums[i % len(nums)])
    

        return resultList
        '''

        return nums + nums